eox_tagging/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.29 on 2020-06-18 19:35
+# Generated by Django 1.11.29 on 2020-06-29 13:35
```

Comment:
please don't modify previous migrations files which have already been executed

Reply:
I added a new migration for the changes. Thank you!